components/Balance/Balance.module.css

```diff
@@ -1,5 +1,5 @@
 .container {
-  height: 60px;
+  height: 75px;
```

Comment:
Can you size the container with responsive sizing? Perhaps a percentage of the page or rem instead of px?